bowlingScorecard.test.js

```diff
@@ -0,0 +1,42 @@
+const BowlingScorecard = require('./bowlingScorecard');
+
+describe('bowlingScorecard', () => {
+  
+  it('returns total score', () => {
+  const myScorecard = new BowlingScorecard()
+  expect(myScorecard.getTotal()).toBe(0)
+  });
```

Comment:
if you're creating new instance of a class in each test you could write before first it this piece of code.
let myScorecard;
beforeEach(() => {
myScorecard = new BowlingScorecard()
};
This will create new instance of a class in each test and will save you time on writing it in each test.